### PR TITLE
Remove City autocomplete feature

### PR DESCRIPTION
**Pull request by B-Galati**

### CODE REVIEW

File: src/Address/Address.php [hunk 1]
```diff
@@ -31,13 +31,14 @@ class Address implements AddressInterface, GeocodableInterface
 
     /**
      * @Assert\Length(max=15, groups={"Default", "Update"})
-     * @Assert\NotBlank(groups={"Update"})
+     * @Assert\Expression(expression="this.getCountry() !== 'FR' or value", message="common.city_name.not_blank", groups={"Update"})
+     *
```
Comment (Remg) [inline src/Address/Address.php]: extra line

Comment (B-Galati) [reply src/Address/Address.php]: fixed

File: src/Address/Address.php [hunk 2]
```diff
@@ -31,13 +31,13 @@ class Address implements AddressInterface, GeocodableInterface
 
     /**
      * @Assert\Length(max=15, groups={"Default", "Update"})
-     * @Assert\NotBlank(groups={"Update"})
+     * @Assert\Expression(expression="this.getCountry() !== 'FR' or value", message="common.city_name.not_blank", groups={"Update"})
```
Comment (Remg) [inline src/Address/Address.php]: should we use the constant `self::FRANCE` for the `FR` value ?

File: src/Address/Address.php [hunk 3]
```diff
@@ -31,13 +31,13 @@ class Address implements AddressInterface, GeocodableInterface
...
      */
     private $city;
 
     /**
      * @Assert\Length(max=255, groups={"Default", "Update"})
-     * @Assert\NotBlank(message="common.city_name.not_blank", groups={"Update"})
+     * @Assert\Expression(expression="this.getCountry() === 'FR' or value", message="common.city_name.not_blank", groups={"Update"})
```
Comment (Remg) [inline src/Address/Address.php]: same here

Comment (B-Galati) [reply src/Address/Address.php]: fixed

File: src/Address/Address.php (hunk 2 above)
Comment (B-Galati) [reply src/Address/Address.php]: fixed

**Pull request merged**